Swiftfin/Views/ItemEditorView/ItemEditorView.swift

```diff
@@ -12,15 +12,30 @@ import SwiftUI
 
 struct ItemEditorView: View {
 
-    @Injected(\.currentUserSession)
-    private var userSession
-
     @EnvironmentObject
     private var router: ItemEditorCoordinator.Router
 
     @ObservedObject
     var viewModel: ItemViewModel
 
+    // MARK: - Can Edit Metadata
+
+    private var canEditMetadata: Bool {
+        viewModel.userSession?.user.permissions.items.canEditMetadata(item: viewModel.item) == true
```

Comment:
These don't need the `?` since it's an implicitly unwrapped optional on the view model.